var/spack/repos/builtin/packages/py-csvkit/package.py

```diff
@@ -34,14 +34,9 @@ class PyCsvkit(Package):
 
     version('0.9.1', '48d78920019d18846933ee969502fff6')
 
-    extends('python')
-
     depends_on('py-dateutil', type=('build', 'run'))
     depends_on('py-dbf', type=('build', 'run'))
     depends_on('py-xlrd', type=('build', 'run'))
-    depends_on('py-sqlalchemy', type=('build', 'run'))
+    depends_on('py-SQLAlchemy', type=('build', 'run'))
```

Comment:
Oops... that needs to be fixed (thought I fixed it).  You are right, it should be `py-sqlalchemy`.